tests/noto_fonts_for_android_test.py

```diff
@@ -52,6 +53,16 @@ def _open_font(font_el) -> ttLib.TTFont:
   return ttLib.TTFont(str(path))
 
 
+def _open_font_path(path) -> ttLib.TTFont:
+  if not path.is_file():
+    raise IOError(f"No such file: {path}")
+
+  if str(path).lower().endswith(".ttc"):
+    # I had no idea how to get fontNumber out of a path :-) so I fixed it at 1
+    return ttLib.TTFont(str(path), fontNumber=1)
```

Comment:
make font_number an argument IMO

Reply:
Or you could do as I did here and just pass the font element in, https://github.com/googlefonts/noto-fonts/blob/main/tests/noto_fonts_for_android_test.py#L45. Actually ... isnt' that the same file? You could probably use that function?